scalatest/src/main/scala/org/scalatest/tools/ScalaTestFramework.scala

```diff
@@ -410,13 +411,15 @@ Tags to include and exclude: -n "CheckinTests FunctionalTests" -l "SlowTests Net
     private def filterMembersOnly(paths: List[String], testClassName: String): Boolean =
       paths.exists(path => testClassName.startsWith(path) && testClassName.substring(path.length ).lastIndexOf('.') <= 0)
       
-    def run(testClassName: String, fingerprint: Fingerprint, eventHandler: EventHandler, args: Array[String]): Unit = {
+    def run(_testClassName: String | Null, fingerprint: Fingerprint | Null, _eventHandler: EventHandler | Null, args: Array[String | Null] | Null): Unit = {
```

Comment:
**Overridden java method arguments are nullable**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.